### ISSUE TITLE
Doesn't work with Ember version 3.18?

### ISSUE THREAD

**Issue by ajstrand** [lifeart/ember-cli-jsx-templates#72]
I'm trying to use this with the latest version of ember, but when I put a simple component in my application.hbs file, Ember tells me it can't find the component when i check my app in the browser. 

my jsx component
```jsx
export default function helloEmber(){
    return <p>hello world from ember!</p>
}

```

my application.hbs

```hbs
<p>this is text</p>
<HelloEmber/>

```

I've tried to fork your library to see if it's an issue I could create a PR for, but so far I've had no success. 

Do you have any ideas why the component wouldn't be to be found in the browser?

Here's the stack trace, if that's helpful. 

```js
opcode-compiler.js:891 Uncaught (in promise) Error: Compile Error: Cannot find component hello-ember
    at IfResolvedComponent (opcode-compiler.js:891)
    at compileOp (opcode-compiler.js:765)
    at pushCompileOp (opcode-compiler.js:727)
    at concatStatements (opcode-compiler.js:1319)
    at compileStatements (opcode-compiler.js:2009)
    at maybeCompile (opcode-compiler.js:1998)
    at CompilableTemplateImpl.compile (opcode-compiler.js:1978)
    at Object.evaluate (runtime.js:3636)
    at AppendOpcodes.evaluate (runtime.js:2004)
    at LowLevelVM.evaluateSyscall (runtime.js:4923)
IfResolvedComponent @ opcode-compiler.js:891
compileOp @ opcode-compiler.js:765
pushCompileOp @ opcode-compiler.js:727
concatStatements @ opcode-compiler.js:1319
compileStatements @ opcode-compiler.js:2009
maybeCompile @ opcode-compiler.js:1998
compile @ opcode-compiler.js:1978
(anonymous) @ runtime.js:3636
evaluate @ runtime.js:2004
evaluateSyscall @ runtime.js:4923
evaluateInner @ runtime.js:4879
evaluateOuter @ runtime.js:4871
next @ runtime.js:5815
next @ runtime.js:5944
RootState.render @ index.js:8771
runInAutotrackingTransaction @ validator.js:106
(anonymous) @ index.js:9069
inTransaction @ runtime.js:1959
_renderRoots @ index.js:9043
_renderRootsTransaction @ index.js:9104
_renderRoot @ index.js:9030
_appendDefinition @ index.js:8945
appendOutletView @ index.js:8931
invoke @ backburner.js:340
flush @ backburner.js:229
flush @ backburner.js:426
_end @ backburner.js:960
Backburner._boundAutorunEnd @ backburner.js:629
Promise.then (async)
iterations @ backburner.js:28
flush @ index.js:43
_scheduleAutorun @ backburner.js:1179
_end @ backburner.js:970
Backburner._boundAutorunEnd @ backburner.js:629
Promise.then (async)
iterations @ backburner.js:28
flush @ index.js:43
_scheduleAutorun @ backburner.js:1179
_end @ backburner.js:970
Backburner._boundAutorunEnd @ backburner.js:629
Promise.then (async)
iterations @ backburner.js:28
flush @ index.js:43
_scheduleAutorun @ backburner.js:1179
_end @ backburner.js:970
Backburner._boundAutorunEnd @ backburner.js:629
Promise.then (async)
iterations @ backburner.js:28
flush @ index.js:43
_scheduleAutorun @ backburner.js:1179
_end @ backburner.js:970
Backburner._boundAutorunEnd @ backburner.js:629
Promise.then (async)
iterations @ backburner.js:28
flush @ index.js:43
_scheduleAutorun @ backburner.js:1179
_ensureInstance @ backburner.js:1167
schedule @ backburner.js:776
schedule @ index.js:361
waitForDOMReady @ application.js:419
init @ application.js:331
superWrapper @ index.js:424
initialize @ core_object.js:98
create @ core_object.js:647
(anonymous) @ app-boot.js:3
```

**Comment by lifeart** [comment]
Hi @ajstrand! Sorry for delayed answer, in general, it should work, if you could create an reproduction repo, it will be great!

**Comment by lifeart** [comment]
@ajstrand  added your case -> https://github.com/lifeart/ember-cli-jsx-templates/blob/be23eb7417ae2efac1dee696ac40f34ff55220d8/tests/dummy/app/components/hello.jsx

**Comment by lifeart** [comment]
published `ember-cli-jsx-templates@0.0.17`
